packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -94,10 +94,15 @@ export class PostPublishPanel extends Component {
 									onSubmit={ this.onSubmit }
 									forceIsDirty={ forceIsDirty }
 									forceIsSaving={ forceIsSaving }
+									isSaving={ isSaving }
```

Comment:
This is calculated here inside the component: https://github.com/WordPress/gutenberg/blob/trunk/packages/editor/src/components/post-publish-button/index.js#L216, no?